### DISCUSSION TITLE
JDK25

### DISCUSSION THREAD

**Discussion by caicai7162** [Meloong-Git/PCL#5221] [提问] [answered]
JDK25为什么无法使用
![7242f27b514125c4e85438f63719abbc](https://github.com/user-attachments/assets/fe72b846-b9df-490b-9ecf-ea978881bed9)

**Comment by Pigeon0v0** [comment]
提交 Issue 并附上启动器日志

**Comment by E0x72-21** [comment]
根据我自己的测试，1.20.4最多只能用jdk22跑，用了jdk23就跑不了了。总之如果没有必要还是老老实实地用jdk21吧

**Comment by LTCatt** [comment, marked as answer]
已作为 bug 修复
